mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -444,13 +444,21 @@ public Mailbox deleteMailbox(MailboxId mailboxId, MailboxSession session) throws
                     .mapToLong(MessageMetaData::getSize)
                     .sum();
 
+                MailboxACL mailboxACL;
+                try {
+                    mailboxACL = listRights(mailbox.generateAssociatedPath(), session);
+                } catch (MailboxException e) {
+                    mailboxACL = new MailboxACL();
+                }
```

Comment:
- [ ] Method extraction to not re-affect variables
 - [ ] Do not swallow exceptions, log them at lest
 - [ ] use the mailboxId to retrieve rights
 
 Why do we need yet another backend read when we can simply `mailbox.getACL()` ?